packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -223,25 +226,89 @@ export function TableNG(props: TableNGProps) {
     const fnBody = `
       const rows = Array(frame.length);
       const values = frame.fields.map(f => f.values);
-
+      let rowCount = 0;
       for (let i = 0; i < frame.length; i++) {
-        rows[i] = {index: i, ${frame.fields.map((field, fieldIdx) => `${JSON.stringify(field.name)}: values[${fieldIdx}][i]`).join(',')}};
+        rows[rowCount] = {____type: 'parent', index: i, ${frame.fields.map((field, fieldIdx) => `${JSON.stringify(field.name)}: values[${fieldIdx}][i]`).join(',')}};
```

Comment:
I like multiple dashes for testing for visibility, great idea 😁

Reply:
_four_ dashes? savage!